src/cloud/reference/cloud-composer.md

```diff
@@ -44,14 +44,22 @@ The metapackage depends on the appropriate versions of `vendor/magento/ece-patch
 
 This package depends on a floating version of `vendor/magento/magento-cloud-configuration` (abbreviated _MCC_). It depends on the major and minor version of MCC that correspond to the specified {{site.data.var.ee}} version, and floats on the patch version so that compatible updates to this packages can be automatically pulled by running `composer update`.
 
-## vendor/magento/ece-tools {#ece-tools}
+The main components of this metapackage are the following :
```

Comment:
Yes, these changes are overdue. This sentence introduces a list. I recommend changing the items below from heading three to list items.

```suggestion
The Cloud metapackage includes the following components:
```

Reply:
I thought about this...technically, they are packages, not components.